### DISCUSSION TITLE
Replace All with Clipboard or Text-Drag

### DISCUSSION THREAD

**Discussion by meteorquake** [WinMerge/winmerge#1597] [Ideas]
1. In a pane you have the Caption line, and there's a menu on the right hand side with Copy, Copy Full Path, etc
It would be useful to have an additional item "Replace with clipboard".
Alternatively (or as well) you could right-click in a pane and have an additional option "Paste Replace All".
This would provide a mouse equivalent of 1. clicking into a pane, 2. Ctrl-A, 3. Ctrl-V
I certainly find I do this task quite a bit, and I would think it would be fairly benign enhancement of use to others who need it a lot. The smoother an action, the less distraction to one's focus.

2. A second version of this which would be useful would be when dragging text into a pane, normally it does a paste, but currently dragging text over the caption does nothing and I think that could helpfully replace the full pane contents with the dragged text.

Cheers, David

**Comment by sdottaka** [comment]
Although not the feature you requested, WinMerge does have a clipboard comparison feature. A demo is available here. https://github.com/WinMerge/winmerge/pull/1147
